pkg/resource/coordinator.go

```diff
@@ -69,6 +69,10 @@ func (c *Coordinator) Coordinate() (ctrl.Result, error) {
 			return result, errors.Wrapf(err, "Check and update finalizer failed")
 		}
 	}
+	err = c.cleanTaskResultMap(f)
```

Comment:
this is already fixed in the release branch, maybe it's better to merge the branch before develop next time

Reply:
OK, I will merge the branch `2.0.0_relase`